<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-VERJL3NI" ignored="0" name="Default test" tests="3" failures="0" timestamp="26 May 2020 22:07:29 GMT" time="0.053" errors="0">
  <testcase name="@BeforeTest setup" time="0.022" classname="com.qa.hubspot.test.LoginPageTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.qa.hubspot.base.Basepage.init_driver(Basepage.java:25)
at com.qa.hubspot.test.LoginPageTest.setup(LoginPageTest.java:25)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setup -->
  <testcase name="@AfterTest doClose" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- @AfterTest doClose -->
  <testcase name="doVerifySignupLinkTest" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- doVerifySignupLinkTest -->
  <testcase name="doVerifyLoginPageTitletest" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- doVerifyLoginPageTitletest -->
  <testcase name="doVerifyLoginTest" time="0.0" classname="com.qa.hubspot.test.LoginPageTest">
    <skipped/>
  </testcase> <!-- doVerifyLoginTest -->
</testsuite> <!-- Default test -->
